src/styles/blocks/navigation.scss

```
@@ -0,0 +1,29 @@
+.nav {
+  &__list {
+    display: flex;
+    font-size: 12px;
+    list-style: none;
+    text-transform: uppercase;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
```

```suggestion
  &__item:not(:last-child) {
      margin-right: 20px;
  }
```

you don't need nesting here 😉